extras/krew.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kadalu
+spec:
+  version: "0.8.2"
```

Comment:
Just a thought (maybe in another PR), wouldn't it better to update `version` and `sha256` automatically as part of release and raise PR against krew-index?

I maybe wrong here, do we need to raise PR against krewindex after updating yaml as part of release?

Reply:
Yes for now it is manual.

> I maybe wrong here, do we need to raise PR against krewindex after updating yaml as part of every release?

Yes by looking at the pull requests https://github.com/kubernetes-sigs/krew-index/pulls?q=is%3Apr+is%3Aclosed